packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -2898,7 +2926,7 @@ class DiagnosticPropertiesBuilder {
   }
 
   /// List of properties accumulated so far.
-  final List<DiagnosticsNode> properties = <DiagnosticsNode>[];
+  List<DiagnosticsNode> properties = <DiagnosticsNode>[];
```

Comment:
this is a scary change. are we sure there's not code that assumes this object is immutable? it seems better to find a way to keep this immutable.